packages/block-library/src/post-author-name/block.json

```diff
@@ -12,11 +12,13 @@
 		},
 		"isLink": {
 			"type": "boolean",
-			"default": false
+			"default": false,
+			"role": "content"
```

Comment:
I think I might move these changes to the `post` blocks to another PR, as they're really not relevant to the topic of the PR.